packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -366,7 +367,7 @@ suite("NoteLookupCommand", function () {
           });
         },
         onInit: async ({ vaults }) => {
-          const cmd = new NoteLookupCommand();
+          const cmd = new NoteLookupCommand(getFakeExtensionForTest());
```

Comment:
nit: could extract a method along the lines of `createNoteLookupCmdForTest/getNoteLookupCmdForTest` that way if there are more changes to the way we do initialization of this command for this test the diff will be just a single line instead of ~30